chapters/002-smol-internet/README.md

```diff
@@ -482,11 +399,11 @@ But then what? `pippin` needs to reply to the ping, `pippin` knows the response
 
 ### Tell `pippin` how to respond to `tara`
 
-We've already seen this in action. At this point, we need to tell `pippin` how to find the `doggonet` network. We did this earlier in teaching `tara` how to find the `squasheeba` network. We're going to leave this as an exercise for the reader to attempt on their own. If you need some guidance, review the [Can `tara` ping `boudi`?](#can-tara-ping-boudi) section.
+We've already seen this in action. At this point, we need to tell `pippin` how to find the `doggonet` network. We did this earlier in teaching `tara` how to find the `squasheeba` network. We're going to leave this as an exercise for the reader to attempt on their own. If you need some guidance, review the [Make `tara` ping hosts on the squasheeba network](#make-tara-ping-hosts-on-the-squasheeba-network) section.
```

Comment:
👍 It strikes me that we should maybe write a markdown linter/tester that verifies that all our links in this document go to an actual file

Reply:
I think the markdown linter I'm already using does that? At least, I remember seeing a yellow squiggle under a broken link. But maybe I'm misremembering that?